### DISCUSSION TITLE
GUI shows no "Pool"-Tab

### DISCUSSION THREAD

**Discussion by liqvideo** [Chia-Network/chia-blockchain#7289] [Support]
Hello,

I updated chia to 1.2.1 (Ubuntu) and the GUI too with the instructions on Github, but the Gui shows no "Pool"-Tab.

I only see:
Full Node
Wallets
Plots
Farm
Keys

What I have to do to get the "Pool"-Tab?

Thanks for your help!

Michael

**Comment by loppefaaret** [comment]
seeing as we have yet to release 1.2.1, i wonder exactly how you updated ? and what is being reported when you do a `chia version` ?

**Reply by liqvideo** [reply]
I updated it 2 times (I thought something went wrong, so I did it a second time) with this instruction:
https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian

Then with the experimental Installer.

"chia version":
1.2.1.dev0

**Reply by loppefaaret** [reply]
ahh, yes, dev0 is a direct mirror of the previous release, so it is in essence 1.2.0 :)
in that case, i suggest you try rebooting, and it should all be fixed

**Reply by liqvideo** [reply]
Had to wait for the plotting to be finished. I restarted the Comp right now - still no luck, my GUI shows no Pool-Tab...

**Reply by loppefaaret** [reply]
if you are using the `git` installation method, please try this:
```
cd ~/chia-blockchain
. ./activate
chia stop all -d
deactivate
cd
rm -rf ~/chia-blockchain
git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules
cd chia-blockchain
sh install.sh
. ./activate
sh install-gui.sh
cd chia-blockchain-gui
npm run electron &
```
that should leave you with the latest release version

**Reply by liqvideo** [reply]
Thank you very much, loppefaaret - it works!

Should I use this script for updating in the future?
